Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-1514.dfy

```diff
@@ -1,4 +1,5 @@
-// RUN: %testDafnyForEachCompiler --refresh-exit-code=0 "%s" -- --standard-libraries --relax-definite-assignment
+// RUN: %verify "%s" --standard-libraries:true  > "%t"
```

Comment:
We definitely shouldn't do this as it's weakening the coverage of these tests quite a bit. Are you still hitting errors in the old version even with the patches now in this PR?

Reply:
Still hitting errors, see e.g. https://github.com/dafny-lang/dafny/actions/runs/7101157152/job/19328811385?pr=4754. Keeping my fix for now.